field.py

```diff
@@ -55,18 +58,29 @@ def populate_field(self):
 
         while bear_counter > 0:
             random_empty_cell = self.get_random_empty_cell()
+
+            if random_empty_cell is None:
+                print >> out_file, "Irrational correlation between field size and amount of bears"
+                sys.exit()
+
             new_bear = create_bear(str(bear_counter), random_empty_cell.row, random_empty_cell.col)
             list_of_bears.append(new_bear)
             random_empty_cell.set_bear(new_bear)
             bear_counter -= 1
 
     def get_random_empty_cell(self):
-        col = randint(0, constants.MAX_COL-1)
-        row = randint(0, constants.MAX_ROW-1)
+        col = randint(0, constants.MAX_COL - 1)
+        row = randint(0, constants.MAX_ROW - 1)
+
+        count_cells = constants.MAX_ROW * constants.MAX_COL
```

Comment:
Interesting idea, but you might randomize the same cell twice
Think of how you can make sure you only choose each cell once, randomly of course